.devcontainer/ci/devcontainer.json

```diff
@@ -6,9 +6,13 @@
     // Sets the run context to one level up instead of the .devcontainer folder.
     "context": "${localWorkspaceFolder}",
 
-    // Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
-    "image": "ghcr.io/blues/note_c_ci:latest",
-    // "dockerFile": "Dockerfile",
+    // Use pre-built image (faster, but only works on amd64 or with Docker Desktop's platform emulation):
+    // "image": "ghcr.io/blues/note_c_ci:latest",
+    // Use local Dockerfile (works on all platforms, required for Apple Silicon):
+    "dockerFile": "Dockerfile",
+
+    // Force amd64 platform for Apple Silicon compatibility
+    "runArgs": ["--platform=linux/amd64"],
```

Comment:
This should be commented out. It would be the unusual case, since CI/CD uses linux.